pkgs/top-level/python-packages.nix

```diff
@@ -6374,6 +6374,10 @@ in {
 
   plexwebsocket = callPackage ../development/python-modules/plexwebsocket { };
 
+  plfit = toPythonModule (pkgs.plfit.override {
```

Comment:
I'm not sure this is the correct name cause plfit is a different package on PyPI: https://pypi.org/project/plfit/

Reply:
Just to clarify, what igraph depends on is this package: https://github.com/ntamas/plfit

Reply:
> Just to clarify, what igraph depends on is this package: https://github.com/ntamas/plfit

That's the one I packaged.
But there's a Python package of the same name, which could lead to confusion.